docs/02.API-REFERENCE.md

```diff
@@ -3868,6 +3868,121 @@ example (void)
 - [jerry_create_promise](#jerry_create_promise)
 - [jerry_promise_state_t](#jerry_promise_state_t)
 
+## jerry_from_property_descriptor
+
+**Summary**
+
```

Comment:
Please mention that this API function is equivalent to `FromPropertyDescriptor` operation .